component/loki/source/gelf/internal/target/metrics.go

```diff
@@ -0,0 +1,42 @@
+package target
+
+// This code is copied from Promtail. The target package is used to
+// configure and run the targets that can read gelf entries and forward them
+// to other loki components.
+
+import "github.com/prometheus/client_golang/prometheus"
+
+// Metrics holds a set of gelf metrics.
+type Metrics struct {
+	reg prometheus.Registerer
+
+	gelfEntries prometheus.Counter
+	gelfErrors  prometheus.Counter
+}
+
+// NewMetrics creates a new set of gelf metrics. If reg is non-nil, the
+// metrics will be registered.
+func NewMetrics(reg prometheus.Registerer) *Metrics {
+	var m Metrics
+	m.reg = reg
+
+	m.gelfEntries = prometheus.NewCounter(prometheus.CounterOpts{
+		Namespace: "promtail",
+		Name:      "gelf_target_entries_total",
```

Comment:
Let's remove the `promtail` namespace and have the metric names be `loki_source_gelf_*`. WDYT?